spec/support/capybara.rb

```diff
@@ -22,6 +22,11 @@
 
 Capybara.javascript_driver = :selenium_chrome_headless
 
+# Allow Capybara to click a <label> even if its corresponding <input> isn't visible on screen.
+# This needs to be enabled when using custom-styled checkboxes and radios, such as those
+# in the GOV.UK Design System.
+Capybara.automatic_label_click = true
```

Comment:
👀

Reply:
This allows GOV UK checkboxes to work correctly when JS is enabled, thanks to @ollietreend for this one.